Client/src/utils/TokenService.js

```
@@ -0,0 +1,232 @@
+            return Date.now() > parseInt(expiryTime);
+          },
+          
+          /**
+           * Get time remaining until token expiry in milliseconds
+           * @returns {number} Milliseconds until expiry, 0 if expired
+           */
+          getTimeUntilExpiry() {
+            const expiryTime = localStorage.getItem(this.tokenExpiryKey);
+            if (!expiryTime) return 0;
+            
+            const remaining = parseInt(expiryTime) - Date.now();
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)